src/components/molecules/BuySellCoin/BuySellCoin.jsx

```diff
@@ -4,6 +4,38 @@ import { Input } from "antd";
 import "./_BuySellCoin.scss";
 import { useAppProvider } from "../../../context/AppProvider";
 import { TRANSACTION_VALIDITY } from "../../../utils/constants";
+import { useWSCProvider } from "milkomeda-wsc-ui-test-beta";
+import { stringToBigNumber } from "../../../utils/helpers";
+import { ethers } from "ethers";
+
+const cardanoReservecoinAddress =
+  "cc53696f7d40c96f2bca9e2e8fe31905d8207c4106f326f417ec36727452657365727665436f696e";
+
+const cardanoStablecoinAddress =
```

Comment:
[eslint] 
src/components/molecules/BuySellCoin/BuySellCoin.jsx
  Line 8:10:  'stringToBigNumber' is defined but never used                  no-unused-vars
  Line 14:7:  'cardanoStablecoinAddress' is assigned a value but never used  no-unused-vars